### PR TITLE
v1.16.3

### PR DESCRIPTION
**Pull request by daipom**
For fluent-package LTS, we are planning to release v1.16.3 on v1.16 branch.

* #3860

master branch will be released as v1.17.0 in the future.

### CODE REVIEW

**Comment by daipom** [comment]
The fixes included in this LTS (stable) version release are limited to bug fixes only.
Commits that refactor codes/performance or change specifications will be included in 1.17.0.
If there are any requests, please let me know.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # v1.16
 
+## Release v1.16.3 - 2023/xx/xx
```
Comment (ashie) [inline CHANGELOG.md:3]: Please update the date.

Comment (daipom) [reply CHANGELOG.md]: Thanks! Fixed!

**Comment by daipom** [comment]
Sorry, I force-pushed d3cf2e0f95a0ad88b9897197db6c5152310f114f into v1.16 in order to fix the releasing commit name.
(`v1.16.3 (#4335)` => `v1.16.3`)

**Pull request merged**